[2025-10-09T04:51:33.990+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: order_generator.load_orders scheduled__2025-10-09T02:00:00+00:00 [queued]>
[2025-10-09T04:51:34.005+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: order_generator.load_orders scheduled__2025-10-09T02:00:00+00:00 [queued]>
[2025-10-09T04:51:34.007+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2025-10-09T04:51:34.010+0000] {taskinstance.py:1284} INFO - Starting attempt 1 of 2
[2025-10-09T04:51:34.011+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2025-10-09T04:51:34.030+0000] {taskinstance.py:1304} INFO - Executing <Task(PythonOperator): load_orders> on 2025-10-09 02:00:00+00:00
[2025-10-09T04:51:34.044+0000] {standard_task_runner.py:55} INFO - Started process 2566 to run task
[2025-10-09T04:51:34.047+0000] {standard_task_runner.py:82} INFO - Running: ['airflow', 'tasks', 'run', 'order_generator', 'load_orders', 'scheduled__2025-10-09T02:00:00+00:00', '--job-id', '155', '--raw', '--subdir', 'DAGS_FOLDER/order_loader_dag.py', '--cfg-path', '/tmp/tmp35idh2ix']
[2025-10-09T04:51:34.052+0000] {standard_task_runner.py:83} INFO - Job 155: Subtask load_orders
[2025-10-09T04:51:34.177+0000] {task_command.py:389} INFO - Running <TaskInstance: order_generator.load_orders scheduled__2025-10-09T02:00:00+00:00 [running]> on host 81fbc710bbce
[2025-10-09T04:51:34.272+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=order_generator
AIRFLOW_CTX_TASK_ID=load_orders
AIRFLOW_CTX_EXECUTION_DATE=2025-10-09T02:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2025-10-09T02:00:00+00:00
[2025-10-09T04:51:34.315+0000] {logging_mixin.py:137} INFO - Successfully loaded data to orders
[2025-10-09T04:51:34.590+0000] {logging_mixin.py:137} INFO - Successfully loaded data to order_items
[2025-10-09T04:51:34.591+0000] {python.py:177} INFO - Done. Returned value was: None
[2025-10-09T04:51:34.602+0000] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=order_generator, task_id=load_orders, execution_date=20251009T020000, start_date=20251009T045133, end_date=20251009T045134
[2025-10-09T04:51:34.626+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2025-10-09T04:51:34.641+0000] {taskinstance.py:2582} INFO - 0 downstream tasks scheduled from follow-on schedule check
